contracts/CollateralFunds.sol

```diff
@@ -6,9 +6,14 @@ pragma solidity ^0.8.7;
  * @author The Everest team.
  */
 contract Fund {
-    mapping(address => uint256) public amountFundedByAddress;
-    uint256 public totalFunds;
+    mapping(address => uint256) public collateralFundedByAddress;
+    uint256 public totalCollateral ;
     address[] public funders;
+    address controllerAddress;
+
+    constructor(address _controllerAddress) {
```

Comment:
Add method documentation (similar to what has been done with all the other functions).